main/src/cgeo/geocaching/maps/google/v2/GooglePositionAndHistory.java

```diff
@@ -286,15 +286,15 @@ private synchronized void drawHistory() {
                 historyObjs.addPolyline(new PolylineOptions()
                     .addAll(points)
                     .color(0xFFFFFFFF)
-                    .width(3)
+                    .width(DisplayUtils.getThinLineWidth() / 2f)
```

Comment:
Shouldn't this be a separate method that uses a separate constant?

Reply:
resolved